src/CFamily.UnitTests/SystemAbstractions/ProcessWrapperTests.cs

```diff
@@ -125,20 +126,22 @@ public void Execute_ProcessTimesOut_CanBeKilled()
             var exeName = WriteBatchFileForTest(TestContext,
 @"@echo Waiting for keyboard input which will not arrive...
 set /p arg=
+exit -999
 ");
 
             using var processScope = StartProcess(exeName);
             var testSubject = new ProcessWrapper(processScope.Process);
 
             // Should timeout be because the batch file is waiting for input...
-            testSubject.WaitForExit(2000);
+            testSubject.WaitForExit(1000);
```

Comment:
Minor tweak to the test - shouldn't need to wait two seconds, since we know the process shouldn't exit.